src/unity/python/turicreate/test/test_style_transfer.py

```diff
@@ -193,26 +193,37 @@ def test_get_styles_success(self):
         _raise_error_if_not_sframe(model_styles)
         self.assertEqual(len(model_styles), len(style))
 
-    def test_export_coreml(self):
+    def _coreml_python_predict(self, coreml_model, img_fixed):
         from PIL import Image
-        import coremltools
-        filename = tempfile.mkstemp('my_style_transfer.mlmodel')[1]
-        model = self.model
-        model.export_coreml(filename)
-
-        coreml_model = coremltools.models.MLModel(filename)
-        img = self.style_sf[0:1][self.style_feature][0]
-        img_fixed = tc.image_analysis.resize(img, 256, 256, 3)
         pil_img = Image.fromarray(img_fixed.pixel_data)
         if _mac_ver() >= (10, 13):
             index_data = np.zeros(self.num_styles)
             index_data[0] = 1
-            coreml_output = coreml_model.predict({self.content_feature: pil_img, 'index':index_data}, usesCPUOnly = True)
+            coreml_output = coreml_model.predict(
+                {self.content_feature: pil_img, 'index': index_data},
+                usesCPUOnly=True)
             img = next(iter(coreml_output.values()))
             img = np.asarray(img)
             img = img[..., 0:3]
+            return img
 
-            self.assertEqual(img.shape, (256, 256, 3))
+    def test_export_coreml(self):
```

Comment:
It doesn't need to be done as part of this PR, but I think we should also add an Objective-C unit test (which we don't really have infrastructure for yet, but could possibly piggyback on our C/C++ tests). That will test the real CoreML code interface. The Python interface is much more forgiving when it comes to model input/output mismatches in type/shape/validation.